.coafile

```diff
@@ -52,7 +52,7 @@ shell = bash
 # Do not allow the word "coala" to ensure the repository can be generalized out
 # for use by other organizations.
 files = **
-ignore += .coafile, *requirements.txt, .travis.yml, LICENSE, .moban.yaml, .moban.dt/community-gitignore.jj2, public/**, _site/**, .ci/check_moban.sh
+ignore += .coafile, *requirements.txt, .travis.yml, LICENSE, .moban.yaml, .moban.dt/community-gitignore.jj2, public/**, _site/**, .ci/check_moban.sh, **/tests/**
```

Comment:
No.

Stop editing this line.

You have functions which give you the name of the org, which you can use.